cgmanifest.json

```diff
@@ -0,0 +1,1634 @@
+{
+   "Registrations":[
+      {
+         "component": {
+             "type": "git",
+             "git": {
+                 "repositoryUrl": "https://github.com/microsoft/appcenter-sdk-android.git",
+                 "commitHash": "af4f0eecef83a0c6c00b26f9e12d215d0df21351"
+             }
+         }
+     },
+     {
+         "component": {
+             "type": "git",
+             "git": {
+                 "repositoryUrl": "https://github.com/microsoft/appcenter-sdk-apple.git",
+                 "commitHash": "68ef3d10a6e3f578d9f6a13ee65e0326b320f23c"
```

Comment:
Are we going to update that every time?

Reply:
That is not automated. That is to be changed on release. Probably worth mentioning it in wiki somewhere

Reply:
I don't really understand since there is a commit reference here.

Reply:
Yeah, it is supprosed to be updated manually upon bumping version of native SDKs used. I am not sure if that is something that can be reliably automated.

Reply:
Automation can be done separately (and for Unity as well). @Jamminroot please make sure that manually updating in the release instruction for now. Should not be a blocker to merge this PR